### ISSUE TITLE
kika-keyboard compatibility?

### ISSUE THREAD

**Issue by AntiSol** [futo-org/voice-input#80]
Hi, thanks for this app, the voice recognition is really great!

I'm using a de-googled android 10 on a unihertz titan (with a physical keyboard). It shows a minimal on-screen keyboard for numbers and symbols using kika-keyboard, but when I try to use its voice input it shows a black screen for a second, then shows me a message saying "no such app found".

I looked in apps -> default -> voice input and I see a single option. It has the futo voice input icon but no label. This option is selected. I can't tell whether the issue lies with kika-keybaord or with the voice input choice.

I'd love to get this working with this keyboard. Thanks!

**Comment by johnnynoone** [comment]
Is this a duplicate of #32?

**Comment by AntiSol** [comment]
Thanks for getting back to me.

uuuh I think probably? my voice input does look similar to the screenshot on #32. I think it's very possible that it would start working if futo supported the speech recogniser api. One comment mentioned that maybe it falls back to this if the google service is not found, this also sounds reasonable to me.

TLDR: I don't really know but I feel like solving those issues might also fix my problem.
